### PR TITLE
Restore Grid setClassNameGenerator example

### PR DESCRIPTION
**Pull request by anasmi**
Fixes #193

### CODE REVIEW

**Comment by anasmi** [comment]
Once this is closed, the https://github.com/vaadin/vaadin-grid-flow/pull/899 pull request can be closed

**Comment by anasmi** [comment]
@hongduc-phan why you closed this pull request?

File: vaadin-grid-flow-demo/src/main/java/com/vaadin/flow/component/grid/demo/GridDemo.java [hunk 1]
```diff
@@ -20,6 +20,7 @@
 import java.util.WeakHashMap;
 import java.util.stream.Collectors;
 
+import com.vaadin.flow.component.Html;
```
Comment (hongduc-phan) [inline vaadin-grid-flow-demo/src/main/java/com/vaadin/flow/component/grid/demo/GridDemo.java]: I think it is better if you move the import to below the import org....

Comment (hongduc-phan) [inline vaadin-grid-flow-demo/src/main/java/com/vaadin/flow/component/grid/demo/GridDemo.java]: I think it is better if you move the import to below org import

Comment (anasmi) [reply vaadin-grid-flow-demo/src/main/java/com/vaadin/flow/component/grid/demo/GridDemo.java]: Done

**Pull request merged**